### PR TITLE
Added a script for removing invalid cases on prod env

### PR DESCRIPTION
**Pull request by ymao2**
## Description

This PR is to add a script to clear up invalid cases from non-prod env
## Self-review checklist

* [ ] (1) Quick stakeholder demo done OR
* [ ] (2) ...bug with before and after screenshots
* [X] (3) Tests passing
* [X] (4) Branch ready to be merged (not work in progress)
* [X] (5) No superfluous changes in diff
* [X] (6) No TODO's without new ticket numbers
* [X] (7) PR Prefixed with ticket number e.g. `CT-7654 ...`
* [X] (8) Data migration script is created if any of letter templates is changed

### Screenshots

https://dsdmoj.atlassian.net/jira/software/c/projects/CT/boards/25?modal=detail&selectedIssue=CT-3990

### CODE REVIEW

File: lib/tasks/remove_invalid_cases.rake [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+          end
+        end
+      end
+      puts "Totally #{counter} invalid cases have been removed."
+    end
+
+    private 
+
+    def is_on_production?
+      ENV['ENV'].present? && ENV['ENV'] == 'prod'
```
Comment (willmcb) [inline lib/tasks/remove_invalid_cases.rake:54]: Doesn't everything on the test environments run in 'prod' mode? Would it not be better to flag based on the host instead like in `Settings.yml` with the feature flags 🤔

**Review by willmcb — APPROVED** [review]
LGTM one small commit though 👍

File: lib/tasks/remove_invalid_cases.rake (hunk 1 above)
Comment (ymao2) [reply lib/tasks/remove_invalid_cases.rake:54]: I think you may get it mixed with another environment variable RAILS_ENV,   which has 2 values 'development' and 'production' and they are for rails's own running mode.   but  ENV['ENV'] is set from our config.yaml which is about which cloud environment this app is in and has values of 'staginig', 'production' and 'development' etc.

Comment (willmcb) [reply lib/tasks/remove_invalid_cases.rake:54]: Yep thats it 🤦

**Pull request merged**